CreeDictionary/CreeDictionary/templates/CreeDictionary/word-entries.html

```diff
@@ -31,7 +31,15 @@ <h2 class="definition-title definition-title--search-result">
                                         data-cy="linguistic-breakdown">{% if result.linguistic_breakdown_head %}
                                     {{ result.linguistic_breakdown_head|join:" + " }}
                                     + {% endif %}
-                                    <strong>{{ result.lemma_wordform.text }}</strong> + {{ result.linguistic_breakdown_tail|join:" + " }}</span>
+                                    <strong>{{ result.lemma_wordform.text|add:":" }}</strong>
+                                    <ol class="linguistic-breakdown">                                    
+                                        {% for result in result.linguistic_breakdown_tail %}
+                                            <li class="linguistic-breakdown__breakdown-tail" data-cy="linguistic-breakdown__breakdown-tail">
+                                                {{ result }}
+                                            </li>
+                                        {% endfor %}
```

Comment:
Please rename the `result` variable throughout this loop. I think a better name would be `linguistic_tag` or `explanation`; `result` is a bit too generic.

Reply:
I ended up going with `linguistic_tag` – thanks!